src/calendar/tests/unit/MonthPicker.ts

```diff
@@ -306,11 +307,11 @@ registerSuite({
 
 	'Change month radios'() {
 		let currentMonth = testDate.getMonth();
-		let closed = false;
+		const onRequestClose = sinon.spy();
 		widget.setProperties({
 			...requiredProps,
-			open: !closed,
-			onRequestClose: () => { closed = true; },
+			open: true,
+			onRequestClose,
 			onRequestMonthChange: (month: number) => { currentMonth = month; }
```

Comment:
I think this can be a `sinon.spy` too, and then test `assert.strictEqual(onRequestMonthChange.args[0], 6)` (I think). Same for the `'Previous/next month buttons'()` test